R/fill.R

```diff
@@ -6,7 +6,12 @@
 #'
 #' Missing values are replaced in atomic vectors; `NULL`s are replaced in lists.
 #'
-#' @inheritParams gather
+#' @section Grouped data frames:
+#' With grouped data frames created by [dplyr::group_by()], `fill()` will be
+#' applied _within_ each group, meaning that it won't fill across group
+#' boundaries.
+#'
+#' @param data A data frame.
```

Comment:
I also made this minor change to avoid inheriting `data` docs from the deprecated `gather()` function